Upgrade ds-pause

d-xo opened the pull request:
- Upgrade `ds-pause` to the latest version
- Integrate the breaking changes
- Minor naming / formatting improvements

asymmetric commented on src/govActions.sol:
```diff
@@ -13,7 +13,7 @@ contract Setter {
 }
 
-contract Plan {
+contract GovActions {
```
`Book`?

gbalabasquer replied:
I agree `Plan` is confusing, I think `Book` as well. They are actions for a proxy.
Not sure if there is a proper 4 letter meme for it.

gbalabasquer commented:
@xwvvvvwx in order to merge this PR and integrate it to the testchain scripts I'll need to have some function to modify the delay. Are we going to add it to the `pause` or is there any reason to add that?

gbalabasquer commented:
@xwvvvvwx any comment on this? If we can not add the `setDelay` function, we will need to review the whole process. So the `pause` should be deployed directly in the scripts and not here.

asymmetric commented:
@gbalabasquer apart from having to deploy the pauses from the scripts, are there actual blockers if the `delay` can't be changed?

I think the issue with changing the `delay` is that we don't know if there are scheduled plans whose semantics change when the `delay` is changed.

For the purposes of a running system, in which we want to change the `delay` of a pause, it seems like it would be better to deploy a new one, and make the existing one read-only (i.e. no new plans can be plotted).

gbalabasquer commented:
I guess it is possible, but yeah as I said, we should remove the pause integration from `dss-deploy` and put something in the middle (like the ds-proxy used to be), then adding a pause and doing all the set up in the scripts which there should be in another contract/spell (if we want to apply all the auth changes atomically).

> I think the issue with changing the `delay` is that we don't know if there are scheduled plans whose semantics change when the `delay` is changed.

Btw this is not the case. The `eta` is defined in `plot`, so a change in the delay of the pause doesn't matter for those already in the pipeline.

asymmetric commented:
Oh, you’re totally right. Then I don’t think I see other reasons not to support changing the `delay`.

gbalabasquer commented:
Thanks @xwvvvvwx !!!

d-xo committed:
dapp upgrade ds-pause

d-xo committed:
integrate latest changes to ds-pause

d-xo committed:
Pause related naming and formatting

- Plan -> GovActions: plan is already used with a different meaning within the pause
- formatting
